.github/mergify.yml

```diff
@@ -118,6 +118,40 @@ pull_request_rules:
   actions:
     merge:
       method: merge
+      # The dict of GitHub user ID and sign-off was built from
+      # https://github.com/instructlab/community/blob/main/MAINTAINERS.md
+      # Then, the sign-off was looked up in the git history
+      commit_message_template: |
+        {{ title }} (#{{ number }})
+
+        {{ body }}
+
+        {% set approved_by = {
+            "alimaredia": "Ali Maredia <amaredia@redhat.com>",
+            "alinaryan": "Alina Ryan <aliryan@redhat.com>",
+            "bjhargrave": "BJ Hargrave <hargrave@us.ibm.com>",
+            "booxter": "Ihar Hrachyshka <ihrachys@redhat.com>",
+            "cdoern": "Charlie Doern <cdoern@redhat.com>",
+            "danmcp": "Dan McPherson <dmcphers@redhat.com>",
+            "derekhiggins": "Derek Higgins <derekh@redhat.com>",
+            "hickeyma": "Martin Hickey <martin.hickey@ie.ibm.com>",
+            "jaideepr97": "Jaideep Rao <jrao@redhat.com>",
+            "joesepi": "Joe Sepi <sepi@joesepi.com>",
+            "kelbrown20": "Kelly Brown <kelbrown@redhat.com>",
+            "khaledsulayman": "Khaled Sulayman <ksulayma@redhat.com>",
+            "leseb": "Sébastien Han <seb@redhat.com>",
+            "mairin": "Máirín Duffy <duffy@redhat.com>",
+            "markmc": "Mark McLoughlin <markmc@redhat.com>",
+            "markstur": "Mark Sturdevant <mark.sturdevant@ibm.com>",
+            "nathan-weinberg": "Nathan Weinberg <nweinber@redhat.com>",
+            "RobotSail": "Oleg S <97077423+RobotSail@users.noreply.github.com>",
```

Comment:
@RobotSail is this your typical email?

Reply:
@nathan-weinberg Yes, I use the GitHub email obfuscation